source/whats-new.txt

```diff
@@ -39,6 +40,21 @@ Learn what's new by version:
 * :ref:`Version 1.1 <kafka-connector-whats-new-1.1>`
 * :ref:`Version 1.0 <kafka-connector-whats-new-1.0>`
 
+.. _kafka-connector-whats-new-1.13:
+
+What's New in 1.13
+------------------
+
+- Added a custom authentication provider interface to support AWS IAM
+  authentication for Source and Sink Connectors.
+
+.. TODO add link To learn more, see the :ref:`` guide.
+
+- Fixes an issue that occurred when validating configuration for Source
```

Comment:
S for consistency: It seems like the rest of the page uses "fixed XYZ" instead of "fixes"

```suggestion
- Fixed an issue that occurred when validating configuration for Source
```